src/index.js

```diff
@@ -8,6 +8,7 @@ window.onload = () => {
   const categories = new Categories(meals, buttons, popupComments);
 
   popupComments.querySelector('#btnClosePopup').addEventListener('click', () => {
+    document.querySelector('.popup').style.display = 'none';
```

Comment:
I kindly suggest you to remove this line of code, since with the .close() method of the dialog tag it already hides the popup, so there's no extra styling needed for this to work.